Reorg of the table of contents

suearkinunity opened the pull request:
### Purpose of this PR

This PR:

* Removes the Menu page
* Reorders most other pages

It doesn't reflect the work done on the Actions Reference area in DOCATT-5849, where I also messed with that area and broke it down to chunks. PR 561, currently in draft, shows that. In this PR, I just removed all those pages so I don't have to deal with them in two PRs.

### Links

**Jira:** DOCATT-6821

### Comments to Reviewers

This isn't finalised - I didn't check links, landing pages etc. I wanted to agree on a TOC first.

suearkinunity committed:
Jon's table update

suearkinunity committed:
Edge_Bevel edit

1. Edited the page
2. Update the action name on the landing page

suearkinunity committed:
No more bevel face as a standalone?

I don't think we have bevel faces as a standalone tool anymore. I think it's bevel edges, with the face edit mode selected.

suearkinunity committed:
Bridge edges

Updated Bridge Edges and clarified (I hope) what open edges mean.

suearkinunity committed:
Bridge edges

Updated

suearkinunity committed:
Replacing example in Bevel Edges

Since you can now bevel faces from the same tool, I replaced the example

suearkinunity committed:
Update Edge_Bevel.md

Replaced the example because this tool now does two things

suearkinunity committed:
Merge branch 'suearkinunity-DOCATT-5849' of https://github.com/Unity-Technologies/com.unity.probuilder into suearkinunity-DOCATT-5849

suearkinunity committed:
Quick push as backup

suearkinunity committed:
Found a new action

suearkinunity committed:
Starting to come to grips with the table

suearkinunity committed:
Getting ready for a PR

suearkinunity committed:
Title fix

suearkinunity committed:
Grow Selection page and image

Plus two small edits to other pages

suearkinunity committed:
First attempt

Reorder the whole thing

modrimkus-unity committed:
Merge branch 'master' into suearkinunity-DOCATT-5849

suearkinunity committed:
Minor change

suearkinunity committed:
Trying a whole new thing

unity-cla-assistant commented:
[![CLA assistant check](https://cla-assistant.cds.internal.unity3d.com/pull/badge/not_signed)](https://cla-assistant.cds.internal.unity3d.com/Unity-Technologies/com.unity.probuilder?pullRequest=562) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla-assistant.cds.internal.unity3d.com/Unity-Technologies/com.unity.probuilder?pullRequest=562) before we can accept your contribution.<br/>**1** out of **2** committers have signed the CLA.<br/><br/>:white_check_mark: modrimkus-unity<br/>:x: suearkinunity<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.cds.internal.unity3d.com/check/Unity-Technologies/com.unity.probuilder?pullRequest=562) it.</sub>

suearkinunity committed:
Select Hole update

suearkinunity committed:
Fill Hole

Merged into one page.

suearkinunity committed:
Having fun with the ring/loop options

My thinking was that a user won't know whether they need a loop or ring selection until they've read the docs. Therefore, those two options need to be directly compared.

suearkinunity committed:
Didn't add the pages

suearkinunity committed:
Very, very bad TOC

suearkinunity committed:
Forgot to create that file

suearkinunity committed:
Don't need to keep that

suearkinunity committed:
Holding this for a moment

suearkinunity committed:
Changing TOC so the docs build

suearkinunity committed:
Bit of a change

suearkinunity committed:
Bit more reorg

Not sure we need the reference pages for some of those windows

suearkinunity committed:
Emptying the original section

Some of those are guesses, but they'll do for now. I'll tidy it up again later.
Not sure the Shift feature still exists; asked about it.

suearkinunity committed:
That was wrong

suearkinunity committed:
Merge pull request #563 from Unity-Technologies/suarkin-DOCATT-5849-4

Action ref PR #4 - merging into my other branch, not Master

suearkinunity committed:
Merge pull request #561 from Unity-Technologies/suearkin-DOCATT-5849-3

Action ref PR #3 - merging into my other branch, *not* into Master

suearkinunity committed:
Update Documentation~/CenterPivot.md

suearkinunity committed:
Merge branch 'suearkinunity-DOCATT-5849' into suearking-DOCATT-5849-2

suearkinunity committed:
No Shift feature

Gabe confirmed that yesterday

suearkinunity committed:
Moving ProBuilderize out to Creating

Gabe approved

suearkinunity committed:
Moving ProBuilderize to Create

Gabe approved

suearkinunity committed:
Horrible title

Not fixing the actual page yet, though.

suearkinunity committed:
Update TableOfContents.md

Making it match Msater

suearkinunity committed:
Basically stashing without stashing

suearkinunity committed:
Select vertex color

But I have some questions

suearkinunity committed:
Tidying up a bit

suearkinunity committed:
Fixing nesting in TOC

suearkinunity committed:
Tidying up TOC

I was using a couple of wrong files.

Also, I'm killing the Tools vs Actions page. First of all, we can't keep calling them actions because Actions is a category of Actions, and we can't do that to our poor users. Second, this page duplicates information that's better presented in its own context, such as creating meshes and the so-called actions. The conceptual difference between tool and action is really not something users need to worry themselves with.

suearkinunity committed:
Extrude Faces update

Got rid of the inset page, because it's just a way to use Extrude

suearkinunity committed:
Lightmap UVs action doesn't exist

So I removed it

suearkinunity committed:
Removing notes to self

suearkinunity committed:
Update TableOfContents.md

Changing to match 5835's changes to the edit pages, which include merging the tips/common page.

suearkinunity committed:
Moving out the MeshFilter component page

It was wrongly nested

JonMUnity commented on Documentation~/workflow-uv-landing.md:
```diff
@@ -0,0 +1,26 @@
+# Materials, Shaders, Textures, and UVs
```
This page still needs to be added to the TOC.

suearkinunity replied:
Replaced by workflow-texture-mapping... not sure why I ended up with three files that have exactly the same content

suearkinunity replied:
Replaced by workflow-texture-mapping... not sure why I ended up with three files that have exactly the same content

JonMUnity reviewed (commented):
TOC looks good to me. I don't have any issues with it.

suearkinunity committed:
Merge branch 'master' into suearkin-DOCATT-6821

suearkinunity commented:
Something here is very odd. I'm going to have to rebuild this locally and see what's what.

suearkinunity committed:
Update Content/Resources/Materials/FacePicker.mat

suearkinunity committed:
Update Face_Extrude.md

Tidying table

suearkinunity committed:
Merge branch 'suearking-DOCATT-5849-2' into suearkin-DOCATT-5849-5

suearkinunity committed:
Merge pull request #568 from Unity-Technologies/suearkin-DOCATT-5849-5

Actions ref PR 5

suearkinunity committed:
Update ProBuilderDefault.mat

Undoing changes in the MAT files

suearkinunity committed:
Merge pull request #560 from Unity-Technologies/suearking-DOCATT-5849-2

Action Reference - PR #2

suearkinunity committed:
Merge branch 'master' into suearkinunity-DOCATT-5849

suearkinunity committed:
Merge branch 'master' into suearkin-DOCATT-6821

suearkinunity committed:
Update Object_ProBuilderize.md

Resolving merge conflict

suearkinunity committed:
Delete Documentation~/ref_actions.md

I don't need this file - I'm about to merge the Actions ref PRs into this PR

suearkinunity committed:
Update TableOfContents.md

Resolving merge conflicts

suearkinunity committed:
Update TableOfContents.md

Resolving merge conflicts

suearkinunity committed:
Merge pull request #555 from Unity-Technologies/suearkinunity-DOCATT-5849

DO NOT MERGE YET: Action Reference - PR #1

suearkinunity commented:
All TOC changes - including all the action ref changes - are now in this PR. 
So this PR is *not* ready for review anymore; I need to test it locally and see that, for example, the TOC is still buildable.

suearkinunity committed:
I have not backed up in way too long
